Gemfile.lock

```diff
@@ -108,49 +108,49 @@ GEM
     mimemagic (0.3.5)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.14.1)
+    minitest (5.14.2)
     msgpack (1.3.3)
     nio4r (2.5.4)
     nokogiri (1.10.10)
       mini_portile2 (~> 2.4.0)
     popper_js (1.16.0)
-    public_suffix (4.0.5)
+    public_suffix (4.0.6)
```

Comment:
### [public_suffix](https://simonecarletti.com/code/publicsuffix-ruby)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 4.0.5 → 4.0.6

[_[change-log](https://github.com/weppos/publicsuffix-ruby/blob/master/CHANGELOG.md), [source-code](https://github.com/weppos/publicsuffix-ruby/tree/v4.0.6)_]





<details>
<summary>Commits</summary>

A change of **6** commits. See the full changes on [the compare page](https://github.com/weppos/publicsuffix-ruby/compare/v4.0.5...v4.0.6).

These are the individual commits:
- (c101bd5) [Fix key duplication in codecov.yml (#175)](https://github.com/weppos/publicsuffix-ruby/commit/c101bd5ef6da1d400e7995ba09372291c73ae203)
- (89c74b0) [Updated definitions](https://github.com/weppos/publicsuffix-ruby/commit/89c74b0446e2d267750b2c58307547fed6ff94c9)
- (c61d25e) [Update rubocop](https://github.com/weppos/publicsuffix-ruby/commit/c61d25e6e3155a879a0536706e7a7e62806ba1f9)
- (e3eedfd) [Ignore rubocop cop](https://github.com/weppos/publicsuffix-ruby/commit/e3eedfd7cc06573ae85d8597dd4bd46c893d851d)
- (a302119) [Last release with 2.3](https://github.com/weppos/publicsuffix-ruby/commit/a302119cd731ab490f321abfa7a2f9b168ad55cf)
- (2e0bee3) [Release 4.0.6](https://github.com/weppos/publicsuffix-ruby/commit/2e0bee34176f902b416796d6dbc5a03c670e0a98)


</details>